src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -118,17 +117,37 @@ public static void usernameLogin(LoginData loginData) {
     }
 
     public static void usernameLogin(LoginData loginData, String domain) {
-        String loginUrl = Http.create("https://" + domain + ".darkorbit.com/")
-                .consumeInputStream(LoginUtils::getLoginUrl);
+        String frontPage = "";
+        String gResponse = "";
+        try {
+            frontPage = IOUtils.read(Http.create("https://" + domain + ".darkorbit.com/").getInputStream());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
 
+        try {
+            CaptchaAPI captcha = CaptchaAPI.getInstance();
+            gResponse = captcha.solveCaptcha(frontPage);
+        }catch (Exception e){
+            e.printStackTrace();
+        }
```

Comment:
```suggestion
    Map<String, String> extraPostParams = Collections.emptyMap();
    CaptchaAPI solver = CaptchaAPI.getInstance();
    if (solver != null) {
        try {
            extraPostParams = solver.solveCaptcha(frontPage);
        } catch (Exception e) {
            System.out.printLn("Captcha solver failed to resolve login captcha");
            e.printStackTrace();
        }
    }
```
Only run if the captcha solver exists, and make sure to say a bit more in console